src/priyanka_Panat/assignment25/StringWithNum.java

```diff
@@ -0,0 +1,44 @@
+/*
+Assignment - 25 : 18th April'2021
+
+Program 2: Write a method which will return all String from given array which contains atleast one number. Main method should print the output.
+Input : {"TechnoCredits", "HP1w", "Pd3R4K9", "Krishna", "Aashvi25"}
+Output : HP1w Pd3R4K9 Aashvi25
+*/
+
+package priyanka_Panat.assignment25;
+
+public class StringWithNum {
+
+	String[] findStringWithNum(String[] str) {
+		String[] result = new String[str.length];
+		int stringIndex = 0;
+		for (int index = 0; index < str.length; index++) {
+			boolean isDigit = false;
+			for (int innerIndex = 0; innerIndex < str[index].length(); innerIndex++) {
+				char ch = str[index].charAt(innerIndex);
+				if (Character.isDigit(ch)) {
+					isDigit = true;
+					break;
+				}
+			}
+			if (isDigit == true)
```

Comment:
if this boolean variable isDigit comes false then still you are increasing stringindex whcih is wrong

Reply:
think on this